frontend/src/types/Common.ts

```diff
@@ -0,0 +1,5 @@
+// Export for Select dropdown type
+export type SelectOption = {
```

Comment:
Why did you change things from `{ value: number; label: string }` to `{ id: string; label: string }`? Does the new Select component only allow strings?

Reply:
Yes, the new Select component requires strings.